src/reactviews/pages/QueryResult/table/plugins/cellSelectionModel.plugin.ts

```diff
@@ -419,6 +427,76 @@ export class CellSelectionModel<T extends Slick.SlickData>
         }
     }
 
+    private async setSelectionSummaryText(isSelection?: boolean) {
```

Comment:
Please add a test for this function!  If you extract this to a helper method that takes grid as an argument, should be easy.